sources/europe/lt/Lithuania-ORT10LT.geojson

```diff
@@ -9,7 +9,7 @@
         "name": "Lithuania - NŽT ORT10LT",
         "url": "http://mapproxy.openmap.lt/ort10lt/g/{z}/{x}/{y}.jpeg",
         "max_zoom": 18,
-        "min_zoom": 4,
+        "min_zoom": 1,
```

Comment:
FYI: I'm getting mostly HTTP/500 (Internal Server Error) exceptions for tiles at zoom levels 1 and 2…